content/en/users/aai/check-in/vos/_index.md

```diff
@@ -68,17 +68,7 @@ A step-by-step guide for the VO registration process is provided in the procedur
 
    ![VO population](./check-in-my-vo-population-result.png)
 
-### Accepting new VO members
-
-Users can request membership in your VO by following the VO enrollment URL. The
-enrollment URL has the following form:
-
-`https://aai.egi.eu/registry/co_petitions/start/coef:##` where `##` is the
-unique numeric identifier for the enrollment flow of your VO.
-
-Once a user submits a VO membership petition, all VO managers are notified with
-an email containing a link to the petition. Any of the VO managers can then
-review the petition and either **approve** or **deny** the request.
+### VO enrollment URLs
```

Comment:
The way in which the enrollment URL can be used is not well understood by our VO Managers, could we spend some other options here to describe how it is used? For example:

- the VO Manager can insert it in an invitation email to be sent directly to the users identified by him.

- It can be used in any registration web page as a href of a button or image etc.

- It can be found by the user himself who has just registered in Check-in via the Enrollment menu
-  others?

Reply:
Hi @vardizzo-lab  I have made some improvements based on your comments.
Can you please review again?

Reply:
OK